### ISSUE TITLE
pondering

### ISSUE THREAD

**Issue by vondele** [glinscott/fishtest#503]
It would be useful to have fishtest support optionally pondering.

Changes should be not too invasive I think. Basically, verify that the number of available cores is nthreads * ( 1 + pondering), and adjust cutechess concurrency, passing the pondering option to cutechess.

**Comment by snicolet** [comment]
Indeed, if a worker gives 8 cores to fishtest, a ponder test with 1 thread games should play only 4 games concurrently because white and black should play on independant cores.
